setup/src/Magento/Setup/Controller/Navigation.php

```diff
@@ -57,7 +59,7 @@ public function __construct(NavModel $navigation, Status $status, ObjectManagerP
      */
     public function indexAction()
     {
-        $json = new JsonModel;
+        $json = new JsonModel();
```

Comment:
Why are you instantiating classes with `new` instead of using `Factory` or just Dependency Injection?

Reply:
Fixed